scripts/inc.buildhelper.sh

```diff
@@ -78,7 +78,7 @@ buildapp(){
 			if [ "$versionname" = "$baseversionname" ]; then
 				density=$(basename "$(dirname "$dpivariant")")
 				buildapk "$dpivariant" "$ziplocation/$density/$targetlocation"
-				echo -n " $density"
```

Comment:
Yes, I believe so. To not write the dpi attached to the versionnumber or a previous dpi.